java/com/google/copybara/git/GitDestination.java

```diff
@@ -549,7 +549,11 @@ public ImmutableList<DestinationEffect> write(TransformResult transformResult,
 
       console.progress("Git Destination: Adding all files");
       try (ProfilerTask ignored = generalOptions.profiler().start("add_files")) {
-        alternate.add().force().renormalize().all().run();
+        if(generalOptions.isForced()){
+          alternate.add().force().all().run();
+        }else{
+          alternate.add().force().renormalize().all().run();
+        }
```

Comment:
We discovered that, on initial sync of a branch w/ `--force` flag usage in global options, adding `--renormalize` results in a situation where no new files are added to that initial destination commit. Hence the conditional. Perhaps this isn't the best design; if so, we request reviewer's input on what can be a better design.